app/backend/src/couchers/errors.py

```diff
@@ -67,6 +67,7 @@
 GROUP_OR_COMMUNITY_NOT_FOUND = "Group or community not found."
 HOST_REQUEST_CLOSED = "This host request is closed, use a normal message instead."
 HOST_REQUEST_IN_PAST = "This host request is in the past."
+HOST_REQUEST_SPAM_USER_BAN = "Due to excessive host requests, the account has been suspended until further notice. The moderation team has been notified."
```

Comment:
Ah, this doesn't suspend the account, right? It just stops you from sending more reqs?

Maybe change it to `You've sent too many host requests in the 24 hours, try again later.`

Reply:
In this case it would ban the user, so it would stop the user indefinitely until a mod review